### PR TITLE
Dashboard Changes

### PR DESCRIPTION
**Pull request by NadaaFarook**
1 - Refactored the step card ,to make the edge case handling better.
2 - Added links from interview step card to admission form
3 - Added cap to resubmission more than 3 times

### CODE REVIEW

File: client/src/components/StepCard/StepcardPayment.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  //   if (status.status == 'portfolio_not_submitted') {
+  //     portfolioSubmissionstatus = 'Not Started'
+  //   } else if (
+  //     status.status == 'under review' ||
+  //     status.status == 'portfolio_needs_revision'
+  //   ) {
+  //     portfolioSubmissionstatus = 'In Progress'
+  //   } else {
+  //     portfolioSubmissionstatus = 'Completed'
+  //   }
```
Comment (SAYUK09) [inline client/src/components/StepCard/StepcardPayment.tsx:34]: Let clean this if not needed

File: client/src/components/StepCard/StepcardPayment.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      </Heading>
+      <a>
+        {
+          <HiArrowCircleRight
+            color={theme.colors.black['700']}
+            style={{ height: '30px', width: '30px' }}
+          />
+        }
+      </a>
+      {/* {status.level == step.level ? (
```
Comment (SAYUK09) [inline client/src/components/StepCard/StepcardPayment.tsx:69]: Same here, we can clean this up

**Review by SAYUK09 — CHANGES_REQUESTED** [review]
Everything seems fine to me.

**Review by MehulKChaudhari — APPROVED** [review]
LGTM 🔥

**Pull request merged**